src/main/java/com/softserve/academy/spaced/repetition/controller/CourseController.java

```diff
@@ -56,102 +55,102 @@
         Page<CourseLinkDTO> courseLinkDTOS = courseService
                 .getPageWithCourses(pageNumber, sortBy, ascending).map((course) -> {
                     Link selfLink = linkTo(methodOn(CourseController.class)
-                            .getCourseById(course.getCategory().getId(), course.getId())).withSelfRel();
+                            .getCourseById(course.getId())).withSelfRel();
                     return DTOBuilder.buildDtoForEntity(course, CourseLinkDTO.class, selfLink);
                 });
         return new ResponseEntity<>(courseLinkDTOS, HttpStatus.OK);
     }
 
-    @GetMapping(value = "/api/courses/ordered")
+    @GetMapping(value = "/api/courses/orders")
     public ResponseEntity<List<CourseLinkDTO>> getAllCoursesOrderByRating() {
         List<Course> courseList = courseService.getAllOrderedCourses();
-        Link collectionLink = linkTo(methodOn(CourseController.class).getAllCoursesOrderByRating()).withRel("course");
+        Link collectionLink = linkTo(methodOn(CourseController.class).getAllCoursesOrderByRating()).withRel("courses");
```

Comment:
This case is similar to previous one. We get list of courses, each course has a link to self and this link we can get from method `getCourseById`.